cmd/container-use/stdio_test.go

```diff
@@ -344,13 +383,14 @@ func TestRepositoryContention(t *testing.T) {
 	}
 
 	const numServers = 10
-	sharedRepoDir, err := os.MkdirTemp("", "cu-e2e-shared-repo-*")
+	testName := strings.ReplaceAll(t.Name(), "/", "_")
+	sharedRepoDir, err := os.MkdirTemp("", fmt.Sprintf("cu-e2e-%s-repo-*", testName))
 	require.NoError(t, err)
 	defer os.RemoveAll(sharedRepoDir)
 
 	setupGitRepo(t, sharedRepoDir)
 
-	sharedConfigDir, err := os.MkdirTemp("", "cu-e2e-shared-config-*")
+	sharedConfigDir, err := os.MkdirTemp("", fmt.Sprintf("cu-e2e-%s-config-*", testName))
```

Comment:
this makes me think you were hitting real bugs and fixed them by making the test less aggro... it's fine to make the test less aggro, but what were the bugs?